src/js/core/widget/core/Popup.js

```diff
@@ -659,6 +714,8 @@
 				eventUtils.off(self._ui.page, "pagebeforehide", self, false);
 				eventUtils.off(window, "resize", self, false);
 				eventUtils.off(document, "vclick", self, false);
+
+				self._onScroll(null);
```

Comment:
instead of this line,
`eventUtils.off(self._ui.content, "scroll", self, false);`

Reply:
done.